frontend/src/pages/projects/screens/detail/ProjectDetails.stories.tsx

```diff
@@ -0,0 +1,116 @@
+import { mockServingRuntimeK8sResource } from '~/__mocks__/mockServingRuntimeK8sResource';
+import { mockServingRuntimesConfig } from '~/__mocks__/mockServingRuntimesConfig';
+import { mockProjectK8sResource } from '~/__mocks__/mockProjectK8sResource';
+import { mockPVCK8sResource } from '~/__mocks__/mockPVCK8sResource';
+import useDetectUser from '~/utilities/useDetectUser';
+import ProjectDetails from './ProjectDetails';
+
+const handlers = (isEmpty: boolean): RestHandler<MockedRequest<DefaultBodyType>>[] => [
+  rest.get('/api/k8s/api/v1/namespaces/test-project/pods', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockPodK8sResource({})]))),
+  ),
+  rest.get(
+    '/api/k8s/apis/route.openshift.io/v1/namespaces/test-project/routes/test-notebook',
+    (req, res, ctx) => res(ctx.json(mockRouteK8sResource({}))),
+  ),
+  rest.get('/api/k8s/apis/kubeflow.org/v1/namespaces/test-project/notebooks', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockNotebookK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/apis/project.openshift.io/v1/projects', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockProjectK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/api/v1/namespaces/test-project/persistentvolumeclaims', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockPVCK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/apis/project.openshift.io/v1/projects/test-project', (req, res, ctx) =>
+    res(ctx.json(mockProjectK8sResource({}))),
+  ),
+  rest.get(
+    'api/k8s/apis/serving.kserve.io/v1beta1/namespaces/test-project/inferenceservices',
+    (req, res, ctx) =>
+      res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockInferenceServiceK8sResource({})]))),
+  ),
+  rest.get('/api/k8s/api/v1/namespaces/test-project/secrets', (req, res, ctx) =>
+    res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockSecretK8sResource({})]))),
+  ),
+  rest.get(
+    'api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes',
+    (req, res, ctx) =>
+      res(ctx.json(mockK8sResourceList(isEmpty ? [] : [mockServingRuntimeK8sResource({})]))),
+  ),
+  rest.get(
+    '/api/k8s/apis/serving.kserve.io/v1alpha1/namespaces/test-project/servingruntimes/test-model',
+    (req, res, ctx) => res(ctx.json(mockServingRuntimeK8sResource({}))),
+  ),
+  rest.get('/api/k8s/api/v1/configmaps/servingruntimes-config', (req, res, ctx) =>
+    res(ctx.json(mockServingRuntimesConfig({}))),
+  ),
+];
+
+export default {
+  title: 'ProjectDetails',
+  component: ProjectDetails,
+  parameters: {
+    reactRouter: {
+      routePath: '/projects/:namespace/*',
+      routeParams: { namespace: 'test-project' },
+    },
+    msw: {
+      handlers: handlers(false),
+    },
+  },
+} as ComponentMeta<typeof ProjectDetails>;
+
+const Template: ComponentStory<typeof ProjectDetails> = (args) => {
+  useDetectUser();
```

Comment:
🤔 This probably should be at the top of the test infra.

cc @Gkrumbach07

Reply:
Yes, it should. I think we can add it to the top at preview.js. We put it here just to get it working quickly without having to figure out how to get it working at the top level. The issue is that that hook requires the context provider to be  above it. we can add it to the global decorator, but that would sit above it. We could create a component just for and place the hook in there and put that under the provider, but that seems like a lot.

Reply:
However, the decorators is an array. so we should be able to just add another below it

Reply:
Gotcha -- so you'll follow up with another issue?

Reply:
I think the best way is to follow up this in another issue because we need to mock up the object that returns from the `status` endpoint.